packages/docusaurus/src/commands/start.ts

```diff
@@ -187,17 +186,17 @@ export default async function start(
       // Reduce log verbosity, see https://github.com/facebook/docusaurus/pull/5420#issuecomment-906613105
       stats: 'summary',
     },
-    static: {
+    static: siteConfig.staticDirectories.map((dir) => ({
```

Comment:
oh, hopefully they allow an array here 😅

Reply:
Yeah, they do:
https://webpack.js.org/configuration/dev-server/#directory
https://webpack.js.org/plugins/copy-webpack-plugin/#root

The one thing I'm _slightly_ worried about though is path collisions. I don't know if any warning should be given in that case